createDoctorWhoDB.sql

```diff
@@ -0,0 +1,135 @@
+CREATE TABLE [tblAuthor]
+(
+ [AuthorId]   int NOT NULL ,
+ [AuthorName] varchar(50) NOT NULL ,
+
+
+ CONSTRAINT [PK_tblauthor] PRIMARY KEY CLUSTERED ([AuthorId] ASC)
+);
+GO
+
+CREATE TABLE [tblCompanion]
+(
+ [CompanionId]   int NOT NULL ,
+ [CompanionName] varchar(50) NOT NULL ,
+ [WhoPlayed]     varchar(50) NOT NULL ,
+
+
+ CONSTRAINT [PK_tblcompanion] PRIMARY KEY CLUSTERED ([CompanionId] ASC)
+);
+GO
+
+CREATE TABLE [tblDoctor]
+(
+ [DoctorId]         int NOT NULL ,
+ [DoctorNumber]     int NOT NULL ,
+ [DoctorName]       varchar(50) NOT NULL ,
+ [BirthDate]        date NOT NULL ,
+ [FirstEpisodeDate] date NOT NULL ,
```

Comment:
i dont think first and last eps dates should be NOT NULL. You can create doctors that have no appearances right ?